### PR TITLE
Fxi #1776: Support HtmlParser in FaqSingleActivity

### LINKED ISSUE #1776: FAQ Single should support HTMLParser
Currently FAQSingleActivity does not support HTML parser because of which the bullet points are not looking good. Make sure that the `answertext` uses HTMLParser.

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

Fixes #1776 

FAQSingle now supports HTMLParser, so that the bullets can be parsed correctly.

## Before
Screenshot_1599554588

## After
Screenshot_1599555318

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/java/org/oppia/app/help/faq/faqsingle/FAQSingleActivityPresenter.kt [hunk 1]
```diff
@@ -37,12 +40,19 @@ class FAQSingleActivityPresenter @Inject constructor(
...
-      Html.fromHtml(answer)
-    }
+
+    // NOTE: Here entityType and entityId can be anything as it will actually not get used.
+    // They are needed only for cases where rich-text contains images from server and in faq
+    // we do not have images.
+    val answerTextView = activity.findViewById<TextView>(R.id.faq_answer_text_view)
+    answerTextView.text = htmlParserFactory.create(
+      resourceBucketName,
+      /* entityType= */ "faq",
```
Comment (anandwana001) [inline app/src/main/java/org/oppia/app/help/faq/faqsingle/FAQSingleActivityPresenter.kt]: as per our earlier discussion, should we go with named argument or this comment argument is fine?

File: app/src/sharedTest/java/org/oppia/app/faq/FAQSingleActivityTest.kt [hunk 2]
```diff
@@ -95,6 +122,24 @@ class FAQSingleActivityTest {
...
   }
 
+  @Test
+  fun openFAQSingleActivity_checkAnswer_isCorrectlyParsed() {
+    val answerTextView = activityTestRule.activity.findViewById(
+      R.id.faq_answer_text_view
+    ) as TextView
+    val htmlParser = htmlParserFactory.create(
+      resourceBucketName,
+      /* entityType= */ "",
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/app/faq/FAQSingleActivityTest.kt]: should we add some comments here why we are sending empty strings for suture reference?

**Review by anandwana001 — APPROVED** [review]
LGTM, Thanks @rt4914 
Added a few thoughts to discuss.

File: app/src/main/java/org/oppia/app/help/faq/faqsingle/FAQSingleActivityPresenter.kt (hunk 1 above)
Comment (rt4914) [reply app/src/main/java/org/oppia/app/help/faq/faqsingle/FAQSingleActivityPresenter.kt]: Done.

File: app/src/sharedTest/java/org/oppia/app/faq/FAQSingleActivityTest.kt (hunk 2 above)
Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/app/faq/FAQSingleActivityTest.kt]: Done.

**Review by anandwana001 — APPROVED** [review]
LGTM, Thanks @rt4914

**Pull request merged**